2024-03-02 21:42:16,429 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-02 21:42:16,429 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Configure stats pid to 35817
2024-03-02 21:42:16,429 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from /home/zungwooker/.config/wandb/settings
2024-03-02 21:42:16,429 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/settings
2024-03-02 21:42:16,429 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_init.py:_log_setup():526] Logging user logs to /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/run-20240302_214216-h97u5wu1/logs/debug.log
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_init.py:_log_setup():527] Logging internal logs to /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/run-20240302_214216-h97u5wu1/logs/debug-internal.log
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8468af2640>
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_init.py:init():566] calling init triggers
2024-03-02 21:42:16,430 INFO    MainThread:35817 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'random seed': 0, 'learning_rate': 0.0001, 'batch_size': 64, 'epochs': 100, 'note': ''}
2024-03-02 21:42:16,431 INFO    MainThread:35817 [wandb_init.py:init():616] starting backend
2024-03-02 21:42:16,431 INFO    MainThread:35817 [wandb_init.py:init():620] setting up manager
2024-03-02 21:42:16,433 INFO    MainThread:35817 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-03-02 21:42:16,437 INFO    MainThread:35817 [wandb_init.py:init():628] backend started and connected
2024-03-02 21:42:16,453 INFO    MainThread:35817 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-02 21:42:16,454 INFO    MainThread:35817 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-02 21:42:16,454 INFO    MainThread:35817 [wandb_init.py:init():720] updated telemetry
2024-03-02 21:42:16,469 INFO    MainThread:35817 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-02 21:42:17,034 INFO    MainThread:35817 [wandb_run.py:_on_init():2262] communicating current version
2024-03-02 21:42:17,137 INFO    MainThread:35817 [wandb_run.py:_on_init():2271] got version response 
2024-03-02 21:42:17,138 INFO    MainThread:35817 [wandb_init.py:init():804] starting run threads in backend
2024-03-02 21:42:19,148 WARNING MainThread:35817 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1176, in init
    run = wi.init()
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 817, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/zungwooker/anaconda3/envs/gene/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-03-02 21:42:28,835 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-02 21:42:28,835 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Configure stats pid to 35817
2024-03-02 21:42:28,835 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from /home/zungwooker/.config/wandb/settings
2024-03-02 21:42:28,835 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/settings
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_init.py:_log_setup():526] Logging user logs to /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/run-20240302_214228-ivnjc4op/logs/debug.log
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_init.py:_log_setup():527] Logging internal logs to /home/zungwooker/gene-cft-2/preprocess/PBA/wandb/run-20240302_214228-ivnjc4op/logs/debug-internal.log
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_init.py:init():566] calling init triggers
2024-03-02 21:42:28,836 INFO    MainThread:35817 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'random seed': 0, 'learning_rate': 0.0001, 'batch_size': 64, 'epochs': 2, 'note': ''}
2024-03-02 21:42:28,837 INFO    MainThread:35817 [wandb_init.py:init():616] starting backend
2024-03-02 21:42:28,837 INFO    MainThread:35817 [wandb_init.py:init():620] setting up manager
2024-03-02 21:42:28,839 INFO    MainThread:35817 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-03-02 21:42:28,840 INFO    MainThread:35817 [wandb_init.py:init():628] backend started and connected
2024-03-02 21:42:28,856 INFO    MainThread:35817 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-02 21:42:28,856 INFO    MainThread:35817 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-02 21:42:28,856 INFO    MainThread:35817 [wandb_init.py:init():720] updated telemetry
2024-03-02 21:42:28,857 INFO    MainThread:35817 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
